src/models/oidc/Providers.ts

```diff
@@ -0,0 +1,31 @@
+export interface BaseOidcProvider {
```

Comment:
These are extractions from other services. I assume we have a reference repository ticket somewhere to some day set up a joint models package?

Reply:
Yes! The idea was to generate the types based on the swagger files some day. Same goes for the swagger files themselves, generating them based on the types/validators